modules/builds-disabling-build-strategy-globally.adoc

```diff
@@ -60,7 +60,7 @@ $ oc edit clusterrole admin
 $ oc edit clusterrole edit
 ----
 
-. For each role, remove the line that corresponds to the resource of the strategy to disable.
+. For each role, specify the sub-resources that correspond to the resource of the strategy to disable.
```

Comment:
```suggestion
. For each role, specify the sub resources that correspond to the resource of the strategy to disable.
```

I don't think we should be using a hyphen for sub-resources. Most uses of "sub-" and "super-" should be prepended without using a hyphen. However, feel free to get a second opinion on this.